pkg/config/dynamic/http_config.go

```diff
@@ -208,6 +210,8 @@ type ServersTransport struct {
 	Certificates        tls.Certificates    `description:"Certificates for mTLS." json:"certificates,omitempty" toml:"certificates,omitempty" yaml:"certificates,omitempty" export:"true"`
 	MaxIdleConnsPerHost int                 `description:"If non-zero, controls the maximum idle (keep-alive) to keep per-host. If zero, DefaultMaxIdleConnsPerHost is used" json:"maxIdleConnsPerHost,omitempty" toml:"maxIdleConnsPerHost,omitempty" yaml:"maxIdleConnsPerHost,omitempty" export:"true"`
 	ForwardingTimeouts  *ForwardingTimeouts `description:"Timeouts for requests forwarded to the backend servers." json:"forwardingTimeouts,omitempty" toml:"forwardingTimeouts,omitempty" yaml:"forwardingTimeouts,omitempty" export:"true"`
+	// VerifyPeerCertificate is only (optionally) set by the Consul Catalog provider in order to implement the Consul Connect protocol between Traefik and the Consul services. Therefore it is not serializable to prevent it from being configurable by other parties.
```

Comment:
Sorry, I forgot to tell you to format it.

```suggestion
	// VerifyPeerCertificate is only (optionally) set by the Consul Catalog provider in
	// order to implement the Consul Connect protocol between Traefik and the Consul
	// services. Therefore it is not serializable to prevent it from being configurable
	// by other parties.
```